Package.swift

```diff
@@ -103,23 +104,23 @@ if let pod_archive_path = ProcessInfo.processInfo.environment["ORT_IOS_POD_LOCAL
     package.targets.append(Target.binaryTarget(name: "onnxruntime", path: pod_archive_path))
 
 } else {
-    // ORT 1.15.0 release
+    // ORT 1.16.0 release
     package.targets.append(
        Target.binaryTarget(name: "onnxruntime",
-                           url: "https://onnxruntimepackages.z14.web.core.windows.net/pod-archive-onnxruntime-c-1.15.0.zip",
-                           checksum: "9b41412329a73d7d298b1d94ab40ae9adb65cb84f132054073bc82515b4f5f82")
+                           url: "https://onnxruntimepackages.z14.web.core.windows.net/pod-archive-onnxruntime-c-1.16.0.zip",
+                           checksum: "684f317081d6795e5fd619972bc5dd9a648156ba9d3e0fb2292314582a216d8e")
     )
 }
 
 if let ext_pod_archive_path = ProcessInfo.processInfo.environment["ORT_EXTENSIONS_IOS_POD_LOCAL_PATH"] {
     package.targets.append(Target.binaryTarget(name: "onnxruntime_extensions", path: ext_pod_archive_path))
 }
-// Note: ORT Extensions 0.8.0 release version pod (Currently not working - it gives a header path not found error.)
+// Note: ORT Extensions latest release version pod currently is not working (it gives a header path not found error.)
```

Comment:
I think onnxruntime-extensions-c 0.9.0 is released now. could we update this target as well?

Reply:
yes. updated as well.